docs/sphinx/manual/optimizing_hpx_applications.rst

```diff
@@ -1008,6 +1008,11 @@ system and application performance.
        (see ``<operation``, e.g. ``en`` or ``eceived``) for the specified
        ``<connection_type>``.
 
+       The performance counters are available only if the compile time constant
+       ``HPX_HAVE_PARCELPORT_COUNTERS`` was defined while compiling the |hpx|
+       core library (which is not defined by default, the corresponding cmake
+       configuration constant is ``HPX_WITH_PARCELPORT_COUNTERS``.
```

Comment:
This misses a closing ')'. It looks like we missed that in other places as well. Could you add it everywhere while you're at it, please?

Reply:
Done. Also fix the same issues on the HPX_HAVE_PARCELPORT_MPI comments.